Gemfile.lock

```diff
@@ -450,4 +450,4 @@ DEPENDENCIES
   yard
 
 BUNDLED WITH
-   1.17.3
+   2.1.4
```

Comment:
I don't think you need to be updating the Gemfile for this module, can you please remove this file?

Reply:
reverted

Reply:
confirmed